Makefile

```diff
@@ -117,5 +117,20 @@ gen:
 
 .PHONY: clean
 clean:
-	rm packetbeat || true
-	rm -r packetbeat-$(VERSION) || true
+	-rm packetbeat
+	-rm packetbeat.test
+	-rm -r packetbeat-$(VERSION)
+	-rm -r coverage
+
+# Generates packetbeat.test coverage testing binary
+test-binary:
+	$(GODEP) go test -c -cover -covermode=count -coverpkg ./...
```

Comment:
Does the dependency on $(GOFILES) mean it is only rebuilt in case on of the files changed? Not sure why this replaces the .PHONY part?